packages/flutter/test/cupertino/context_menu_test.dart

```diff
@@ -810,4 +811,71 @@ void main() {
       expect(right.dx, lessThan(left.dx));
     });
   });
+
+  testWidgets('Is gesture conflicts of ctx_menu resolved',
```

Comment:
Nit: What is ctx_menu?  Maybe change this to: `testWidgets('Conflicting gesture detectors',`